.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   authorize:
     name: Authorize
-    runs-on: macos-13
+    runs-on: macos-13-xlarge
```

Comment:
why -xlarge

Reply:
installing visionOS platform requires arm64:
```
Run xcodebuild -downloadPlatform visionOS -quiet
Developing for visionOS requires a Mac with Apple silicon.
Error: Process completed with exit code 70.
```

macos-13 is x86, add -xlarge to make sure arm64
reference: https://github.com/actions/runner-images

Reply:
:+1: